index.php

```diff
@@ -14,11 +14,10 @@
  * @return string of dinos to be displayed
  */
 function checkIfSearched(PDO $db): string {
-    if (isset($_GET['submit'])) {
-        return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search']));
-    } else {
-        return Museum::displayAllDinos(DinosaurHydrator::getAllDinos($db));
+    if (isset($_GET['submit'], $_GET['search'], $_GET['filter'])) {
+        return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search'], $_GET['filter']));
```

Comment:
Could you move this function into it's own file? Because it's not a class, composer won't be able to autoload it. I'd suggest just adding a functions.php into the src directory containing this function as well as `searchedValue` and then just requiring it into this file.

Reply:
That sounds like a plan! :)